lte/gateway/c/session_manager/SessionStore.h

```diff
@@ -56,10 +57,10 @@ struct SessionSearchCriteria {
 
   SessionSearchCriteria(
       const std::string p_imsi, SessionSearchCriteriaType p_type,
-      const uint32_t secondary_key_unit32)
+      const uint32_t secondary_key_unit32v)
```

Comment:
Why the variable name change here?

Reply:
Might not be your change but `unit32` -> `uint32` :D

Reply:
`secondary_key_uint32`

Reply:
clean up/corrected.